lua/zen-mode/view.lua

```diff
@@ -213,17 +213,25 @@ function M.create(opts)
   vim.api.nvim_exec(augroup:format(M.win, M.win), false)
 end
 
-function M.fix_hl(win, normal)
+function M.fix_hl(win, is_bg)
   local cwin = vim.api.nvim_get_current_win()
-  if cwin ~= win then
-    vim.api.nvim_set_current_win(win)
-  end
-  normal = normal or "Normal"
-  vim.cmd("setlocal winhl=NormalFloat:" .. normal)
-  vim.cmd("setlocal winblend=0")
-  vim.cmd([[setlocal fcs=eob:\ ,fold:\ ,vert:\]])
-  -- vim.api.nvim_win_set_option(win, "winhighlight", "NormalFloat:" .. normal)
-  -- vim.api.nvim_win_set_option(win, "fcs", "eob: ")
+  vim.api.nvim_set_current_win(win)
+
+  local_options = {
+    winblend = 0,
+  }
+
+  if is_bg then
+    local_options["winhighlight"] = "NormalFloat:ZenBg"
```

Comment:
technically, we don't need to avoid setting fillchars for the BG window, but you don't need to, and I already wrote this code, so I left it in